src/pyOpenMS/pxds/GNPSMetaValueFile.pxd

```diff
@@ -15,11 +15,6 @@ cdef extern from "<OpenMS/FORMAT/GNPSMetaValueFile.h>" namespace "OpenMS":
         #  
         #  This will produce the minimal required meta values and can be extended manually.
         #  
-        #  Parameters
-        #  
-        #  consensus_map: ConsensusMap
-        #   Input ConsensusMap from which the input mzML files will be determined.
-        #  
-        #  output_file : str
-        #   Output file path for the meta value table.  
+        #  :param consensus_map: Input ConsensusMap from which the input mzML files will be determined.
```

Comment:
Please be consistent with spaces before  colons ":"

Reply:
Yes that is fine. I mean the ":" that ends the param roles.

Reply:
Sorry my bad. Thankyou @jpfeuffer.